stackman-api/src/main.rs

```diff
@@ -101,8 +107,15 @@ fn main() -> Result<()> {
                 let mut albums = holdings
                     .values()
                     .filter(|album| {
-                        if let Some(stack) = request.get_param("stack") {
-                            album.stack == stack
+                        if let Some(collection) = request.get_param("collection") {
+                            album.collection == collection
+                        } else {
+                            true
+                        }
+                    })
+                    .filter(|album| {
+                        if let Some(new) = request.get_param("new") {
+                            new != "true" || album.year == 2022
```

Comment:
On the real API server, this would probably be a field in some database since there's no hard date cutoff for when we remove something from the new music rotation. I can change it to 2023 if you want. I'd need to add some 2023 albums to my test data.